app/assets/javascripts/materialize_effects.js

```diff
@@ -1,8 +1,14 @@
 function setMaterialize(element, component, op) {
-    var elems = document.querySelectorAll(element);
-    var instances = M[component].init(elems, op);
- }
- document.addEventListener('DOMContentLoaded', function() {
-    setMaterialize('.slider', 'Slider', {});
-    setMaterialize('.sidenav', 'Sidenav', {});
- }); 
+   var elems = document.querySelectorAll(element);
+   var instances = M[component].init(elems, op);
+}
+
+document.addEventListener('DOMContentLoaded', function() {
+   setMaterialize('.slider', 'Slider', {});
+   setMaterialize('.sidenav', 'Sidenav', {});
+}); 
+ 
+document.addEventListener("turbolinks:load", function() {
+ M.AutoInit();
+ $(".dropdown-trigger").dropdown();
```

Comment:
Single quotes...
🤔  was jQuery added to the project?

Reply:
my bad, forget to deleted